### ISSUE TITLE
Keine Korrelation zwischen Beleg und zugehörigem Projekt möglich

### ISSUE THREAD

**Issue by davidhommel** [selectline-software/selectline-api#267]
Hallo zusammen,

Wenn ich per GET /Documents... einen Beleg abrufe, bekomme ich dort die ProjectId des ggf. verknüpften Projektes zurückgeliefert.
Anhand dieser ProjectId kann ich aber leider nicht auf das Projekt selbst zugreifen, weil im Pfad /Project die Projektnummer (Number) als Identifikationsmerkmal verwendet wird und die ProjectId nicht ausgegeben oder als Filter angeboten wird. 

Beispiel:
ProjectId als Rückgabe von GET /Documents...: "6"
Number als Identifikation des Projektes in GET /Projects...: "P010"

**Comment by MatthiasGuse** [comment]
Hallo,

das ist korrekt und nachvollziehbar. Ich habe das als Workitem für die Weiterentwicklung angelegt.

Viele Grüße

**Comment by davidhommel** [comment]
Hi,

ist in der Version 23.1 leider immernoch aktuell, schade. Aber ansonsten hat sich echt schon viel getan, supi!

**Comment by davidhommel** [comment]
Gibt es hier eine Neuigkeit zu?

**Comment by MatthiasGuse** [comment]
Nein, bisher gibt es hier keine Änderung. Einen Zeitpunkt kann ich heute ebenfalls noch nicht nennen.

Viele Grüße
